settei/presets/flask.py

```diff
@@ -48,16 +48,28 @@ def before_first_request():
         default=False
     )
 
+    web_on_loaded = config_property(
+        'web.on_loaded', typing.Union[list, str],
+        """Code block str or a sequence of function
+        to execute when the web loaded.""",
+        default=[]
+    )
+
     @cached_property
     def web_config(self) -> typing.Mapping[str, object]:
         """(:class:`typing.Mapping`) The configuration maping for
         web that will go to :attr:`flask.Flask.config <Flask.config>`.
 
         """
-        web_config = self.config.get('web', {})
-        if not isinstance(web_config, collections.abc.Mapping):
+        raw_config = self.config.get('web', {})
+        if isinstance(raw_config, collections.abc.Mapping):
+            web_config = {k.upper(): v for k, v in raw_config.items()}
+        else:
             web_config = {}
-        return ImmutableDict((k.upper(), v) for k, v in web_config.items())
+        web_config.update(
+            ON_LOADED=self.web_on_loaded,
```

Comment:
I think `web_config['ON_LOADED']` is not used.

Reply:
```suggestion
            ON_LOADED=self.web_on_loaded,
```
이게 안쓰이기도 할텐데, `web_config.update(ON_LOADED=self.web_on_loaded)`로 한줄로 써도되지않나요?